.circleci/config.yml

```diff
@@ -22,20 +25,32 @@ jobs:
       - checkout
       - python/install-packages:
           pkg-manager: pip
+          cache-version: << pipeline.parameters.cache-version >>
           app-dir: ~/project/sample_pip
       - run:
           name: "Test"
           working_directory: ~/project/sample_pip
           command: |
             pytest
 
+  pip-install-test-no-packages:
+    executor: python/default
+    steps:
+      - checkout
+      - run:
+          name: "Test"
+          working_directory: ~/project/sample_pip
+          command: |
+            pytest
+
```

Comment:
What is this testing of the orb? Is this meant to have a pip install step?

Reply:
Good catch. I believe I meant to change this out to use the test command, with no requirements file to test the ensure-test-tool script. Will fix!